tests/recipes/SupportedBrowser.html

```diff
@@ -0,0 +1,4 @@
+<html> 
```

Comment:
In both of your tests you're testing the valid case but you're not testing the invalid ones. Make sure to add tests that you don't want to pass (eg: "The knife had sharp edges" - I know includes is case sensitive but let's assume someone capatilized Edges for some reasong :) ).